python/cugraph-nx/run_nx_tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2023, NVIDIA CORPORATION.
+# NETWORKX_GRAPH_CONVERT=cugraph NETWORKX_BACKEND_TEST_EXHAUSTIVE=True pytest --pyargs networkx "$@"
+NETWORKX_TEST_BACKEND=cugraph NETWORKX_TEST_FALLBACK_TO_NX=True pytest --pyargs networkx "$@"
```

Comment:
What's the behavior of `NETWORKX_TEST_FALLBACK_TO_NX`? I'm just wondering if this will silently revert to Nx and pass the test at times when it should be using cugraph and failing because there's a bug we need to fix.

Reply:
Comment added.

Good question. It reverts to networkx if we don't implement a function, or if `can_run` returns False or our algorithm raises `NotImplementedError`. This is helpful to get increased testing and coverage. Other exceptions will raise, so I don't think this will fail silently when we don't want it to. But, we may want to toggle this option during development. Also, it would be nice to ensure our algorithm files have decent coverage when running networkx tests to make sure they are being exercised.

Reply:
ok, my fear (maybe I'm being paranoid) is a bug in our plugin that prevents it from being loaded somehow, and everything falls back to native Nx and silently passes.

If we run without this in dev then that's probably adequate.

Reply:
I'll add `--cov=cugraph_nx/algorithms --cov-report term-missing` to this pytest command to show coverage of algorithms, which will let us know if they run. I agree this is an important sanity check.